homeassistant/components/hassio/system_health.py

```diff
@@ -0,0 +1,54 @@
+"""Provide info to system health."""
+import os
+
+from homeassistant.components import system_health
+from homeassistant.core import HomeAssistant, callback
+
+SUPERVISOR_PING = f"http://{os.environ['HASSIO']}/supervisor/ping"
+OBSERVER_URL = f"http://{os.environ['HASSIO']}:4357"
+
+
+@callback
+def async_register(
+    hass: HomeAssistant, register: system_health.SystemHealthRegistration
+) -> None:
+    """Register system health callbacks."""
+    register.async_register_info(system_health_info, "/hassio")
+
+
+async def system_health_info(hass: HomeAssistant):
+    """Get info for the info page."""
+    info = hass.components.hassio.get_info()
+    host_info = hass.components.hassio.get_host_info()
+    supervisor_info = hass.components.hassio.get_supervisor_info()
+
+    information = {
+        "host_os": host_info.get("operating_system"),
+        "update_channel": info.get("channel"),
+        "supervisor_version": info.get("supervisor"),
+        "docker_version": info.get("docker"),
+        "disk_total": f"{host_info.get('disk_total')} GB",
+        "disk_used": f"{host_info.get('disk_used')} GB",
+        "healthy": supervisor_info.get("healthy"),
```

Comment:
If we're not healthy, could we link here to the page where they can learn more? 

```python
{
  "type": "failed",
  "error": "Unhealthy",
  "more_info": "/hassio/info"
}
```

Reply:
Implemented for `healthy` and `supported`, both points to `/hassio/system` which contains more details